woocommerce/woocommerce-android
# [Tablet support M2] Full flow

Closes: #10568 


### Description

This PR introduces a `nav_host_fragment` within the `OrderListFragment`, facilitating a two-pane layout for tablets. On phones, this layout adjustment will have no impact. The `nav_host_fragment` is designated to host the Order detail screen.

**Note:** This PR solely focuses on layout adjustments and does not encompass any business logic for selecting orders or automatically opening the first order by default.

### Testing instructions

1. Run the app on physical tablet or in a tablet emulator
2. Click on the `Orders` tab
3. Ensure you see a 2 pane layout with order list and order detail
4. Go into split mode and resize the window and ensure the layout moves between 2 pane and single pane layout according to the window size
5. Run the same app on phone and ensure the behaviour is same as before and there's no change

### Images/gif

<img width="859" alt="Screenshot 2024-01-23 at 9 01 26 AM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/e387f094-e772-4837-9883-83ef6a1dad6e">
